enzy_htp/structure/atom.py

```diff
@@ -276,6 +281,26 @@ def is_connected(self) -> bool:
         """check if self is in the connected state"""
         return self._connect is not None
 
+    @property
```

Comment:
Probly move it to the region with all the other @property? I can help on this.

Reply:
Also probably for consistency purpose, we can rename it as key_str as Residue did.